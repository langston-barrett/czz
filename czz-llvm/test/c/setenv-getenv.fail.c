#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
  int success = setenv("FOO", "bar", 0);
  printf("success=%d\n", success);
  printf("FOO=%s\n", getenv("FOO"));
  return 0;
}
