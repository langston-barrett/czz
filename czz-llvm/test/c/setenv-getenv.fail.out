success=0
FOO=bar
